apps/src/applab/dataBrowser/DataOverview.jsx

```diff
@@ -45,7 +45,8 @@ const DataOverview = React.createClass({
       error => {
         if (typeof error === 'string' && (
           error.includes('maximum number of tables') ||
-          error.includes('The table name is invalid')
+          error.includes('The table name is invalid' ||
+          error.includes('The table was renamed'))
```

Comment:
yeesh! fixed, without thinking too hard about how that managed to pass manual testing shown in the screencasts